debian/rules

```diff
@@ -7,3 +7,6 @@
 
 override_dh_auto_test:
 	./runtests pep8 small
+
+override_dh_installman:
+	help2man -o debian/ubuntu-make.1 umake
```

Comment:
Wow, I hadn't even considered that, I feel a bit silly! I'll have a new commit after I shower and eat, and hopefully that'll work